xep-0298.xml

```diff
@@ -30,22 +30,22 @@
       <firstname>Emil</firstname>
       <surname>Ivov</surname>
       <email>emcho@jitsi.org</email>
-      <jid>emcho@sip-communicator.org</jid>
+      <jid>emcho@jit.si</jid>
     </author>
     <author>
       <firstname>Enrico</firstname>
       <surname>Marocco</surname>
       <email>enrico.marocco@telecomitalia.it</email>
-      <jid>enrico@tilab.com </jid>
+      <jid>enrico@tilab.com</jid>
+    </author>
+    <author>
+      <firstname>Saúl Ibarra</firstname>
+      <surname>Corretgé</surname>
+      <email>saul@ag-projects.com</email>
+      <jid>saul@ag-projects.com</jid>
     </author>
     <discuss>jingle</discuss>
     <revision>
-      <version>0.1</version>
```

Comment:
It does! I'll update the PR.
On Jul 2, 2015 19:24, "Matthew A. Miller" <notifications@github.com> wrote:

> In xep-0298.xml <https://github.com/xsf/xeps/pull/8#discussion_r33801310>:
>
> >      </author>
> >      <discuss>jingle</discuss>
> >      <revision>
> > -      <version>0.1</version>
>
> The existing revision cannot be removed; please add it back.
>
> Also, it seems necessary to add a for 0.2. Does the following look
> acceptable?
>
> <revision>
>   <version>0.2</version>
>   <date>2015-07-02</date>
>   <initials>sic</initials>
>   <remark><p>Correcting errors in grammar and examples; aligning closer to dependent specifications.</p></remark>
> </revision>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xsf/xeps/pull/8/files#r33801310>.
>